src/declarations/stencil-public-compiler.ts

```diff
@@ -302,9 +302,18 @@ export interface ConfigExtras {
    * loading components when using a bundler such as Vite or Parcel. Setting this flag to `true` will change how Stencil
    * lazily loads components in a way that works with additional bundlers. Setting this flag to `true` will increase
    * the size of the compiled output. Defaults to `false`.
+   * @deprecated Use `enableImportInjection`
```

Comment:
I'd like us to give folks a little more instruction here. Can we update this to something like:
 ```suggestion
    * @deprecated This flag has been deprecated in favor of `enableImportInjection`, which provides the same
   * functionality. `experimentalImportInjection` will be removed in a future major version of Stencil.
   */
```